keps/prod-readiness/sig-scheduling/3243.yaml

```diff
@@ -0,0 +1,6 @@
+# The KEP must have an approver from the
+# "prod-readiness-approvers" group 
+# of http://git.k8s.io/enhancements/OWNERS_ALIASES
+kep-number: 3243
+alpha:
+  approver: TBD
```

Comment:
I would be happy to take it once it's ready

Reply:
Thanks.